packages/backend/src/machine-utils.ts

```diff
@@ -174,6 +175,18 @@ export function isLinux(): boolean {
   return linux;
 }
 
+export function isMac(): boolean {
+  return env.isMac;
+}
+
+export function isArm(): boolean {
+  return os.arch() === 'arm64';
+}
+
+export function isX86(): boolean {
+  return os.arch() === 'x64';
+}
```

Comment:
maybe file an issue to add arch on the env

Reply:
https://github.com/containers/podman-desktop/issues/9560